salt/backups/backup_tracking.sls

```diff
@@ -0,0 +1,18 @@
+{% set edx_tracking_bucket = 'odl-residential-tracking-data' %}
+{% set edx_tracking_bucket_creds = salt.vault.read('aws-mitx/creds/read-write-{bucket}'.format(bucket=edx_tracking_bucket)) %}
```

Comment:
This isn't going to work because this state file will be compiled on the minion which doesn't have permissions to read from Vault. We will need to pass this value via pillar data, or modify or Vault config to create a policy which permits access to that route and associates the policy with the minion via EC2 auth or approle or something like that.